snuba/datasets/errors_replacer.py

```diff
@@ -151,19 +156,28 @@ def get_project_exclude_groups_key(
     return f"project_exclude_groups:{f'{state_name.value}:' if state_name else ''}{project_id}"
 
 
+def get_project_exclude_groups_replacement_type_key(key: str) -> str:
+    return f"{key}-type"
+
+
 def set_project_exclude_groups(
-    project_id: int, group_ids: Sequence[int], state_name: Optional[ReplacerState]
+    project_id: int,
+    group_ids: Sequence[int],
+    state_name: Optional[ReplacerState],
+    replacement_type: str,
 ) -> None:
     """Add {group_id: now, ...} to the ZSET for each `group_id` to exclude,
     remove outdated entries based on `settings.REPLACER_KEY_TTL`, and expire
     the entire ZSET incase it's rarely touched."""
 
     now = time.time()
     key = get_project_exclude_groups_key(project_id, state_name)
+    type_key = get_project_exclude_groups_replacement_type_key(key)
```

Comment:
nit:

Given that you need both keys to actually do the thing we want to do, maybe it would be better to make `get_project_exclude_groups_key` return a tuple of `group_key, type_key`